src/app/shared/components/cluster-summary/template.html

```diff
@@ -14,7 +14,7 @@
 limitations under the License.
 -->
 
-<div class="km-wizard-summary"
+<div [ngClass]="showNumbering ? 'km-wizard-summary' : 'km-wizard-dialog-summary'"
```

Comment:
This name was not the best before. I would rather not put "wizard" in it at all.